src/value.h

```diff
@@ -28,10 +28,12 @@ typedef enum {
   T_DOUBLE = 0x040,
   T_PTR = 0x080,
   T_CONSTSTRING = 0x100,
+  T_NODE = 0x200,
+  T_EDGE = 0x400,
 
   // special types for +inf and -inf on all types:
-  T_INF = 0x200,
-  T_NEGINF = 0x400,
+  T_INF = 0x800,
```

Comment:
We keep on moving T_INT and T_NEGINF,
maybe we should increase these values to give us some extra gap for future values?

Reply:
Good call! I switched them to 1<<30 and 1<<31.